src/map/battle.cpp

```diff
@@ -6407,15 +6440,18 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 				ATK_ADD(wd.weaponAtk, wd.weaponAtk2, sstatus->matk_min);
 		}
 #endif
+
+int i = 0;
+
+#ifndef RENEWAL
 		// add any miscellaneous player ATK bonuses
 		if( sd && skill_id && (i = pc_skillatk_bonus(sd, skill_id))) {
```

Comment:
Please, remove the unnecessary brackets in this and the other if below

Reply:
done